src/promptflow-evals/promptflow/evals/evaluators/content_safety/hate_unfairness.py

```diff
@@ -30,7 +33,7 @@ def __init__(self, project_scope: dict, credential: Optional[TokenCredential] =
         # Load the flow as function
         current_dir = Path(__file__).resolve().parent
         flow_dir = current_dir / "flow"
-        self._flow = load_flow(source=flow_dir)
+        self._flow = load_flow(source=flow_dir, log_level=log_level)

```

Comment:
could we directly set the logger here?

Reply:
Or just give user some doc that they can change logging behavior by setting some env variable.

Reply:
The main problem here is that all the log records, the [task](https://msdata.visualstudio.com/Vienna/_workitems/edit/3096911/) are not coming from flow itself, it is from invoker/executor infrastructure.
We are using two mechanisms of getting loggers. If user will set the `PF_LOGGING_LEVEL`, the flow executor logs will ignore it, because the environment variable will have to be set before importing promptflow, as the logger is initialized during import (see the `logger_utils.py`).
```python
flow_logger = get_logger("execution.flow")
```

Another problem is that we initialize the "flowinvoker" logger on flow invoker level, using different mechanism, which will respect `PF_LOGGING_LEVEL` (the code is given before this PR):
```python
self.logger = kwargs.get("logger", LoggerFactory.get_logger("flowinvoker"))
```
So that the logger created/set on the flow level will not have the effect. All